pgjdbc/src/test/java/org/postgresql/test/jdbc2/OuterJoinSyntaxTest.java

```diff
@@ -0,0 +1,194 @@
+package org.postgresql.test.jdbc2;
+
+import static org.junit.Assert.assertTrue;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.postgresql.test.TestUtil;
+
+/**
+ * Executes the query with the outer join syntax. The joined tables are encapsulated with parenthesis. 
+ * 
+ * <b>Note:</b> This query has worked up to driver version 9.4.1211 . Encapsulation with parenthesis is used by third party
+ * like CrystalReports.  
+ */
+public class OuterJoinSyntaxTest
+{
+  /**
+   * The name of the test first table.
+   */
+  public static String MY_TABLE = "mytable";
+  
+  /**
+   * The name of the test second table.
+   */
+  public static String YOUR_TABLE = "yourtable";
+  
+  /**
+   * The name of the test third table.
+   */
+  public static String OUR_TABLE = "ourtable";
+  
+  /**
+   * The connection to the test database.
+   */
+  private Connection connection;
+  
+  /**
+   * Prepares the test environment. This means:
+   * <ul>
+   * <li>open a connection to the test database</li>
+   * <li>create the tables, that are used by this test</li>
+   * <li>fill data into the test tables</li>
+   * </ul>
+   * 
+   * @throws Exception
+   */
+  @Before
+  public void setUp() 
+  throws Exception 
+  {
+    connection = TestUtil.openDB();
+    
+    TestUtil.createTable(connection, MY_TABLE, "id int not null, text varchar(255) not null");
+    TestUtil.createTable(connection, YOUR_TABLE, "id int not null, text varchar(255) not null");
+    TestUtil.createTable(connection, OUR_TABLE, "id int not null, text varchar(255) not null");
+    
+    fillTestTable(
+        connection, 
+        MY_TABLE, 
+        new Object[][] {
+            {1, MY_TABLE + "text_01"},
```

Comment:
What is the purpose of having all this in the test?
Does the test validate the data? No.

Please remove all the excessive stuff.